data/ny/retired/Assembly-District--39265ac2-b5e7-49fe-ae45-4164ccbc7aa3.yml

```diff
@@ -0,0 +1,22 @@
+id: ocd-person/39265ac2-b5e7-49fe-ae45-4164ccbc7aa3
+name: Assembly District 23
+image: https://assembly.state.ny.us/write/upload/member_files/000/headshot/000.jpg
+party:
```

Comment:
@bfossen-ce note sure what's happening here. Assembly district 23 is held by stacey G. Pheffer Amato - https://ballotpedia.org/Stacey_G._Pheffer_Amato

Reply:
At the time of the scrape this seat was still in a recount. Will delete this record